FirebaseAuth/Sources/AuthProvider/Phone/FIRPhoneAuthProvider.m

```diff
@@ -739,21 +743,43 @@ - (void)reCAPTCHAURLWithEventID:(NSString *)eventID completion:(FIRReCAPTCHAURLC
                                                                                  value:appID]];
                                      }
 
-                                     if (self->_auth.requestConfiguration.languageCode) {
-                                       [queryItems
-                                           addObject:[NSURLQueryItem
-                                                         queryItemWithName:@"hl"
-                                                                     value:self->_auth
-                                                                               .requestConfiguration
-                                                                               .languageCode]];
-                                     }
```

Comment:
Looks like this was accidentally deleted.

Reply:
done thx